types/facebook-js-sdk/index.d.ts

```diff
@@ -476,13 +479,14 @@ declare namespace facebook {
     //
     ////////////////////////
     interface AuthResponse {
-        accessToken: string;
-        data_access_expiration_time: number;
+        accessToken?: string;
```

Comment:
If you're adding `?`, please also add `| undefined` (like the other fields) for `exactOptionalProperties`.

Reply:
That being said, I am certainly curious how many people are depending on these things not being undefined; this is a popular package after all.

Reply:
Hi @jakebailey, I have updated the code accordingly.

> That being said, I am certainly curious how many people are depending on these things not being undefined; this is a popular package after all.

I'm not sure either.

Reply:
This does break the usage of the package for me indeed. Also, can't it be only undefined when used in combination with facebook business? Then I suppose we should have 2 different interfaces, one for personal, one for business.

Reply:
Without something to discriminate between the two, I'm not sure how splitting the interface is going to work; you get back this object in certain places and there's no way to know that you will always have these properties, which is more or less the reason to say that they could be undefined.